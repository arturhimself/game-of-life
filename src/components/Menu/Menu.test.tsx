import React from 'react';
import renderer from 'react-test-renderer';
import { Menu } from './';

describe('Menu', () => {
  const onClick = jest.fn();

  it('render menu', () => {
    expect(
      renderer.create(<Menu handleClick={onClick} />).toJSON()
    ).toMatchSnapshot();
  });
});