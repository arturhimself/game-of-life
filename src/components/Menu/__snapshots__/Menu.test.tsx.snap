// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu render menu 1`] = `
<div
  className="menu"
>
  <button
    onClick={[Function]}
  >
    Start
  </button>
</div>
`;
